### PR TITLE
KNL-1578 Added support for local english translations

### PR DESCRIPTION
**Pull request by jayee**

### CODE REVIEW

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java [hunk 1]
```diff
@@ -779,6 +774,19 @@ public String toString() {
                        '}';
     }
 
+
+	public static class BundleControl extends ResourceBundle.Control {
+		@Override
+		public List<Locale> getCandidateLocales(String baseName, Locale locale) {
+			List<Locale> candidateLocales = super.getCandidateLocales(baseName, locale);
+
+			if ("".equals(locale.getLanguage())) {
```
Comment (buckett) [inline kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java]: According to the Javadoc you shouldn't be adding any other `Locale`s when the requested one is `Locale.ROOT`.

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java [hunk 2]
```diff
@@ -22,14 +22,7 @@
...
 import java.text.MessageFormat;
-import java.util.Collection;
-import java.util.Enumeration;
-import java.util.HashMap;
-import java.util.Locale;
-import java.util.Map;
-import java.util.MissingResourceException;
-import java.util.ResourceBundle;
-import java.util.Set;
+import java.util.*;
```
Comment (buckett) [inline kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java]: Sakai is trying not to use `*` imports, could you re-expand this.

File: kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java (hunk 1 above)
Comment (jayee) [reply kernel/kernel-util/src/main/java/org/sakaiproject/util/ResourceLoader.java]: We use sv_SE as system locale, actual sv_SE_umu to include local swedish translations.
We have en_US and sv_SE_umu as selectable locales in sakai.
If the user selects en_US the locale in sakai is set to "".

**Comment by ern** [comment]
If your making changes to text via properties, please look at using the Message Bundle Manager tool.

**Pull request closed without merging**